apps/api/v2/src/modules/auth/guards/permissions/permissions.guard.spec.ts

```diff
@@ -13,7 +14,7 @@ describe("PermissionsGuard", () => {
 
   beforeEach(async () => {
     reflector = new Reflector();
-    guard = new PermissionsGuard(reflector, createMock<TokensRepository>());
+    guard = new PermissionsGuard(reflector, createMock<TokensRepository>(), createMock<ConfigService>());
```

Comment:
NIT: 

Why not test and mock this the way nestjs suggestes. 

https://docs.nestjs.com/fundamentals/testing#testing-utilities 
e.g. 
``` 
  let guard: PermissionGuard;
  let tokenRepository: TokensRepository;
  let configService: ConfigService;

  beforeEach(async () => {
    const module: TestingModule = await Test.createTestingModule({
      providers: [
        PermissionGuard,
        {
          provide: TokensRepository,
          useValue: {
              /* jest fns can go here */
          },
        },
        {
          provide: ConfigService,
          useValue: {
            get: jest.fn(),
          },
        },
      ],
    }).compile();

    guard = module.get<PermissionGuard>(PermissionGuard);
    tokenRepository = module.get<TokensRepository>(TokensRepository);
    configService = module.get<ConfigService>(ConfigService);
  });
```